### DISCUSSION TITLE
How detach the inner-most session in a nested zellij session

### DISCUSSION THREAD

**Discussion by ylorn** [zellij-org/zellij#2030] [Q&A] [answered]
Say I have a nested session of zellij

```
----------------------------------
| local machine session          |
|    =========================   |
|    | remote machine session |  |
|    =========================   |
----------------------------------
``` 

How can I detach the remote machine's zellij session, without detaching the local machine's?

**Comment by imsnif** [comment, marked as answer]
I think if you do `ctrl-g` to lock the outer session and then ctrl+s + o + d it should do the trick. We definitely want to make this flow more "native" though, I know @har7an has done some thinking about this.

**Reply by ylorn** [reply]
Thanks that works. Looking forward for a more native approach!

**Comment by sourproton** [comment]
I found this discussion while searching for how to do it. Maybe there has been an update, because now the more intuitive `ctrl-g` `ctrl-o` `d` works too. That is, lock the outer session with `ctrl+g`, put the inner in `session` mode with `ctrl-o` will  and detach it with `d`.
